Packs/CheckPointHEC/IncidentTypes/incidenttype-CheckPointHEC_Security_Event.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id": "CheckPointHEC Security Event",
+    "version": -1,
+    "vcShouldIgnore": false,
+    "locked": false,
+    "name": "CheckPointHEC Security Event",
```

Comment:
You might want to add a custom incident layout for this incident type which includes all the new custom incident fields introduced. See [this article](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.8/Cortex-XSOAR-Administrator-Guide/Customize-Incident-Layouts) with an explanation.